### PR TITLE
PATAND-248: Update instruction layout

### PR DESCRIPTION
**Pull request by jaimesDSG**
## Objective
Implement the change for the completion step

## Branches
- PAT: development
- Axon: development
- RS: task/PATAND-248_instruction_step
- Cortex: development

#### Credentials:
- Environment: QA
- Org: hybridstudy
- Study: QA-Regression
- User: matias.battaglia+4@medable.com/qpal1010
- Task: instruction test

#### Steps:
* open flask/ resumed it 
* Login if it's not the case
* Start the right task
Expected: 
* The new UI will fit the new ui (same margin that the other, etc.) There is none zeplin for this one

Close PATAND-248

### CODE REVIEW

**Comment by Gryzor** [comment]
Not sure if this was happening before this refactor but I notice a double "Cancel":

<details>
<summary>Example</summary>

image

</details>

**Comment by jaimesDSG** [comment]
> Not sure if this was happening before this refactor but I notice a double "Cancel":
> 
> Example

I fixed this :)

File: backbone/src/main/java/org/researchstack/backbone/ui/step/layout/InstructionStepLayout.java [hunk 1]
```diff
@@ -84,19 +86,18 @@ public int getContentResourceId() {
...
     private void initializeStep() {
         if (step != null) {
-
             // Set Title
             if (!TextUtils.isEmpty(step.getTitle())) {
-                TextView title = (TextView) findViewById(R.id.rsb_intruction_title);
+                TextView title = findViewById(R.id.rsb_intruction_title);
                 title.setVisibility(View.VISIBLE);
-                title.setText(step.getTitle());
+                title.setText(HtmlCompat.fromHtml(step.getTitle(),HtmlCompat.FROM_HTML_MODE_COMPACT));
```
Comment (Gryzor) [inline backbone/src/main/java/org/researchstack/backbone/ui/step/layout/InstructionStepLayout.java:93]: not important to fix now, but you're missing a space here.

File: backbone/src/main/java/org/researchstack/backbone/ui/step/layout/InstructionStepLayout.java [hunk 2]
```diff
@@ -84,19 +86,18 @@ public int getContentResourceId() {
...
             }
 
             // Set Summary
             if (!TextUtils.isEmpty(step.getText())) {
-                TextView summary = (TextView) findViewById(R.id.rsb_intruction_text);
+                TextView summary = findViewById(R.id.rsb_intruction_text);
                 summary.setVisibility(View.VISIBLE);
-                summary.setText(Html.fromHtml(step.getText()));
+                summary.setText(HtmlCompat.fromHtml(step.getText(),HtmlCompat.FROM_HTML_MODE_COMPACT));
```
Comment (Gryzor) [inline backbone/src/main/java/org/researchstack/backbone/ui/step/layout/InstructionStepLayout.java:100]: and here ;)

Comment (jaimesDSG) [reply backbone/src/main/java/org/researchstack/backbone/ui/step/layout/InstructionStepLayout.java]: done

**Pull request merged**